### PR TITLE
OJ-1216 add metrics to access token handler

### PR DESCRIPTION
**Pull request by scronk1**
### What changed

Add metrics handler to the access token handler lambda

### Issue tracking

- [OJ-1216](https://govukverify.atlassian.net/browse/OJ-1216)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

### Other considerations

- [x] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1216]: https://govukverify.atlassian.net/browse/OJ-1216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jkunle** [comment]
We should ensure that our test calls but metric, in both success and failure paths
Please see if there is any other logging or metrics by having a look at corresponding java lambda https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/main/accesstoken/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AccessTokenHandler.java#L59

File: lambdas/src/handlers/access-token-handler.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class AccessTokenLambda implements LambdaInterface {
             const requestPayload = this.requestValidator.validatePayload(event.body);
             const sessionItem = await this.sessionService.getSessionByAuthorizationCode(requestPayload.code);
             logger.appendKeys({ govuk_signin_journey_id: sessionItem.clientSessionId });
-            logger.info("found session")
+            logger.info("found session");
```
Comment (surajkumar) [inline lambdas/src/handlers/access-token-handler.ts:39]: This should be ideally debug or trace level logging. Consider making the message a bit more informative such as passing in the ID that was found

**Review by surajkumar — CHANGES_REQUESTED** [review]

File: lambdas/src/handlers/access-token-handler.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ const dynamoDbClient = createClient(AwsClientType.DYNAMO) as DynamoDBDocument;
 const ssmClient = createClient(AwsClientType.SSM) as SSMClient;
 const configService = new ConfigService(ssmClient);
 const initPromise = configService.init([CommonConfigKey.SESSION_TABLE_NAME, CommonConfigKey.SESSION_TTL]);
+const ACCESS_TOKEN = "accesstoken";
```
Comment (surajkumar) [inline lambdas/src/handlers/access-token-handler.ts:21]: Typescript convention is camel case for consts

File: lambdas/src/handlers/access-token-handler.ts (hunk 1 above)
Comment (scronk1) [reply lambdas/src/handlers/access-token-handler.ts:39]: I'm following our pattern of implementing as close as possible to the java lambdas, I agree in theory but this is from the java lambda https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/a2c1d73dca85643fe9d120c68d338f830476f5a8/accesstoken/src/main/java/uk/gov/di/ipv/cri/common/api/handler/AccessTokenHandler.java#L59

**Review by surajkumar — APPROVED** [review]

**Comment by jkunle** [comment]
I think we still need the `journey id` https://github.com/alphagov/di-ipv-cri-lib/blob/main/src/main/java/uk/gov/di/ipv/cri/common/library/util/EventProbe.java#L63

**Comment by scronk1** [comment]
> I think we still need the `journey id` https://github.com/alphagov/di-ipv-cri-lib/blob/main/src/main/java/uk/gov/di/ipv/cri/common/library/util/EventProbe.java#L63 on the first chain of the call on eventProbe `.addJourneyIdToLoggingContext(sessionItem.getClientSessionId())`
> 
> ```
>             eventProbe
>                     .addJourneyIdToLoggingContext(sessionItem.getClientSessionId())
>                     .log(Level.INFO, "found session");
> ```

Is this not covered by the existing line before it? https://github.com/alphagov/di-ipv-cri-common-lambdas/blob/87c7bdf28146e905872e8229142c543aa79cab91/lambdas/src/handlers/access-token-handler.ts#L38

**Review by jkunle — APPROVED** [review]

**Pull request merged**